src/sentry/incidents/tasks.py

```diff
@@ -174,6 +174,7 @@ def auto_resolve_snapshot_incidents(alert_rule_id, **kwargs):
     if incidents:
         incidents = incidents[:batch_size]
         for incident in incidents:
+            trigger_incident_triggers(incident)
```

Comment:
We could potentially just have this in `update_incident_status` and change
`if status_method == IncidentStatusMethod.MANUAL and status == IncidentStatus.CLOSED`
 to
`if status_method in (IncidentStatusMethod.MANUAL, IncidentStatusMethod.RULE_UPDATED) and status == IncidentStatus.CLOSED`